app/models/katello/content_override.rb

```diff
@@ -0,0 +1,36 @@
+module Katello
+  class ContentOverride
+    attr_accessor :content_label, :name, :value
+    ENABLEMENT_VALUES = ["yes", "no", "0", "1", "default"].freeze
+
+    def initialize(content_label)
+      @content_label = content_label
+    end
+
+    def enabled=(value = nil)
+      @name = "enabled"
+      @value = value
+    end
+
+    def to_hash
+      ret = {"content_label" => @content_label, "name" => @name}
+      ret["value"] = @value if @value
+      ret
+    end
+
+    def self.create_enabled(content_params)
```

Comment:
Let me put this in perspective, ContentOverride accepts a content_label, name/key to override and value. In Katello so far we have only used one particular value for  'name' be it AK/Content Hosts and that name or key is "enabled".

More over I need a central place to hold the logic  that says parse the content_params being send and create me an "enabled" ContentOverride object from it.

ContentOverride.new(label,{value: 1}) will not tell me that the name value here should be "enabled" . So a better approach is use the new method to set generic content override where you will pass me label, name and value that are not specifically related to setting "enabled" overriding attributes.